addon-test-support/helpers.js

```diff
@@ -189,10 +189,10 @@ export function find(selector, contextEl) {
   } else if (Object.prototype.toString.call(selector) === "[object String]") {
     selected = document.querySelectorAll(`${settings.rootElement} ${selector}`);
   }
-  if (!settings.useJQueryWrapper) {
-    return (selected.length === 1) ? selected[0] : selected;
+  if (settings.useJQueryWrapper) {
```

Comment:
@rwjblue @cibernox ah yeah Web Standards for the win :) I'll yank out the `settings. useJQueryWrapper` option. Standard HTMLElement or NodeList query result from `find` will be great to use. And if a tests needs jQuery it will be more explicit in the test code for the developer to wrap the find call, e.g. `jQuery(find('a'))`.